### PR TITLE
Finalize hand over to new maintainers.

### PR DESCRIPTION
**Pull request by OrangeTux**
I just removed my access the project's Github and Pypi.

Thanks for moving this project for forward @mdwcrft , @jainmohit2001  and  @proelke . Good luck!

@mdwcrft I didn't include your last name, since your Github profile doesn't expose that information. 
Let me know if you want to include that.

cc: @Jared-Newell-Mobility

### CODE REVIEW

**Comment by ajmirsky** [comment]
@OrangeTux it seems that the new code owners haven't had the bandwidth needed in the past few months/weeks to keep this project moving forward; I know how busy this time of year can be for many!  But the project's PR backlog continues to grow; would you consider expanding the code owners group so that the project can keep pace with the community's needs?

I'd rather keep the community's focus on a single repo instead of having it fracture by others relying on other forks which have more current fixes and improvements.

**Comment by proelke** [comment]
We (the maintainers taking over) had a call today to organize ourselves. We plan to approve and merge a lot of the open PRs and cut a release on Monday/Tuesday. Apologies for the radio silence but things are now moving again. We've mapped out a rough roadmap and improvements to examples and documentation.

File: README.rst [hunk 1]
```diff
@@ -203,6 +203,14 @@ To lower the logs for this package only use the following code:
   logging.getLogger('ocpp').setLevel(level=logging.DEBUG)
   logging.getLogger('ocpp').addHandler(logging.StreamHandler())
 
+Aknowledgements
+---------------
+
+Till the end of 2024, this project has been lead and maintained by `Auke Oosterhoff`_ and
+`Jared Newell`_. We thank them for work their work! 
+
+Since than, the the project is lead by `Chad`_, `Mohit Jain`_ and `Patrick Roelke`_.
```
Comment (jainmohit2001) [inline README.rst:212]: There is an extra `the` in this line.

File: README.rst [hunk 2]
```diff
@@ -217,3 +225,9 @@ Attribution-NoDerivatives 4.0 International Public License.
 .. _rtd: https://ocpp.readthedocs.io/en/latest/index.html
 .. _The Mobility House: https://www.mobilityhouse.com/int_en/
 .. _websockets: https://pypi.org/project/websockets/
+
+.. _Auke Oosterhoff:  https://github.com/orangetux
+.. _Jared Newell: https://github.com/Jared-Newell-Mobility
+.. _Chad: https://github.com/mdwcrft
```
Comment (mdwcrft) [inline README.rst:231]:
```suggestion
.. _Chad Meadowcroft: https://github.com/mdwcrft
```

File: README.rst (hunk 1 above)
Comment (mdwcrft) [inline README.rst:212]:
```suggestion
Since than, the project is lead by `Chad Meadowcroft`_, `Mohit Jain`_ and `Patrick Roelke`_.
```

**Review by mdwcrft — APPROVED** [review]

**Pull request merged**